pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -612,14 +620,11 @@ void enrichWithDSMEventDates(Handle handle,
                         .orElse(null);
 
                 if (birthDate != null && aomRule != null) {
-                    LocalDate localBirthDate = birthDate.asLocalDate().orElse(null);
-                    if (localBirthDate != null) {
-                        dateOfMajority = aomRule.getDateOfMajority(localBirthDate);
-                    }
+                    dateOfMajority = aomRule.getDateOfMajority(birthDate);
                 }
             }
 
-            participant.setBirthDate(birthDate.asLocalDate().orElse(null));
```

Comment:
Alerts in Slack are point to this line. If there's no birth date, then it looks like this will throw an NPE.

Reply:
Annoying. Thanks for fixing this.